docs/source/key_components/dashboard_visualization.rst

```diff
@@ -0,0 +1,269 @@
+Dashboard Visualization
+=======================
+
+Env-dashboard is a post-experimental visualization tool, aims to provide
+more intuitive environment information, which will guide the design of
+the algorithm and continually fine-tuning.
+
+Currently, the visualization of senario **Container Inventory Management**
+and **Citi Bike** are supported.
+
+Dependency
+----------
+
+Module **streamlit** and **altair** should be pre-installed.
+
+* `streamlit <https://www.streamlit.io/>`_
+
+
+* `altair <https://altair-viz.github.io/>`_
+
+Install them with:
+
+.. code-block:: sh
+
+    pip install streamlit altair
+
+----
+
+How to Use?
+-----------
+
+Generate dumped data
+~~~~~~~~~~~~~~~~~~~~
+
+The dumped data from environment is the data source of visualization.
+User needs to specify a parameter when creating Env object to determine
+if needs to dump data. To generate data, user needs to specify the
+parameter **options** with following code:
```

Comment:
To dump environment data for visualizing, user needs to ... when initialie the environment instance.